Dockerfile

```
@@ -0,0 +1,21 @@
+FROM python:3-buster as base
+WORKDIR /usr/src/app
+ENV VIRTUAL_ENV=/usr/src/app/venv
+RUN python -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+RUN pip install gunicorn
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+COPY ./poetry.toml ./poetry.toml
+COPY ./poetry.lock ./poetry.lock
+COPY ./pyproject.toml ./pyproject.toml
```

You can copy multiple files in a single command:

```suggestion
COPY ./poetry.toml ./poetry.lock ./pyproject.toml ./
```